src/main/java/hw3/HeaderMenu.java

```diff
@@ -0,0 +1,118 @@
+package hw3;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class HeaderMenu {
+
+    private WebDriver driver;
+
+    @FindBy(linkText = "Home")
+    private WebElement homeButton;
+
+    @FindBy(linkText = "Contact form")
+    private WebElement contactFormButton;
+
+    @FindBy(linkText = "Service")
+    private WebElement serviceButton;
+
+    @FindBy(linkText = "Metals & Colors")
+    private WebElement metalsAndColorsButton;
+
+    @FindBy(name = "main-title")
+    private WebElement mainHeaderText;
+
+    @FindBy(linkText = "JDI GITHUB")
+    WebElement subHeaderText;
+
+    @FindBy(name = "jdi-text")
+    private WebElement lowerMainHeaderText;
+
+    private final String ATTRIBUTEOFSUBHEADERTEXT = "href";
```

Comment:
What is the name style described in the Java Code Convention for the `private final` variables